.cirrus.yml

```diff
@@ -0,0 +1,64 @@
+---
+# We use Cirrus for Vagrant tests, because macOS instances of GHA
+# are too slow and flaky, and Linux instances of GHA do not support KVM.
+
+compute_engine_instance:
+  image_project: cirrus-images
+  image: family/docker-kvm
+  platform: linux
+  nested_virtualization: true
+  # CPU limit: `16 / NTASK`: see https://cirrus-ci.org/faq/#are-there-any-limits
+  cpu: 8
+  # Memory limit: `4GB * NCPU`
+  memory: 32G
+
+task:
+  timeout_in: 30m
+  env:
+    DEBIAN_FRONTEND: noninteractive
+    HOME: /root
+    # yamllint disable rule:key-duplicates
+    matrix:
+      DISTRO: fedora34
+      DISTRO: centos7
+  info_script: |
+    uname -a
+    echo "-----"
+    cat /etc/os-release
+    echo "-----"
+    cat /proc/cpuinfo
+    echo "-----"
+    df -T
+  install_libvirt_vagrant_script: |
+    apt-get update
+    apt-get install -y libvirt-daemon libvirt-daemon-system vagrant vagrant-libvirt
+    systemctl enable --now libvirtd
+  vagrant_cache:
+    fingerprint_script: uname -s ; cat Vagrantfile.$DISTRO
+    folder: /root/.vagrant.d
+  vagrant_up_script: |
+    ln -sf Vagrantfile.$DISTRO Vagrantfile
+    # Retry if it fails (download.fedoraproject.org returns 404 sometimes)
+    vagrant up || vagrant up
+    mkdir -p -m 0700 /root/.ssh
+    vagrant ssh-config >> /root/.ssh/config
+  system_info_script: |
+    ssh default 'sh -exc "uname -a && systemctl --version && df -T && cat /etc/os-release"'
+  unit_tests_script: |
+    ssh default 'sudo -i make -C /vagrant localunittest'
+  integration_systemd_script: |
+    ssh -tt default "sudo -i make -C /vagrant localintegration RUNC_USE_SYSTEMD=yes"
```

Comment:
most probably `-tt` is not needed on Cirrus -- it was added because GHA execution environment is terminal-less.

Reply:
done

Reply:
-tt seems required on Cirrus as well 😞 

```
not ok 28 runc create can chdir if runc has access
# (in test file tests/integration/cwd.bats, line 73)
#   `[ "$status" -eq 0 ]' failed
# runc spec (status=0):
# 
# runc run test_busybox (status=1):
# time="2021-07-15T06:22:45Z" level=error msg="open /dev/tty: no such device or address"
```

https://cirrus-ci.com/task/5495579875737600

Reply:
Ah, OK, good to know.